Source/ApiTemplate/Source/ApiTemplate/ViewModels/Car.cs

```diff
@@ -1,48 +1,49 @@
 namespace ApiTemplate.ViewModels
 {
 #if Swagger
-    using ApiTemplate.ViewModelSchemaFilters;
-    using Swashbuckle.AspNetCore.Annotations;
-
     /// <summary>
     /// A make and model of car.
     /// </summary>
-    [SwaggerSchemaFilter(typeof(CarSchemaFilter))]
 #endif
     public class Car
     {
 #if Swagger
         /// <summary>
         /// The cars unique identifier.
         /// </summary>
+        /// <example>1</example>
```

Comment:
How would you do a complex object using the `<example>` XML node?

Reply:
A complex object would have it's own comments for the properties it owns.